### PR TITLE
Update pacparser recipe: add version 1.3.8

### PR DESCRIPTION
**Pull request by iarspider**

### CODE REVIEW

File: var/spack/repos/builtin/packages/pacparser/package.py [hunk 1]
```diff
@@ -9,9 +9,13 @@
 class Pacparser(MakefilePackage):
     """pacparser is a library to parse proxy auto-config (PAC) files."""
 
+    maintainers = ['iarspider']
+
     homepage = "https://pacparser.github.io/"
     url      = "https://github.com/manugarg/pacparser/releases/download/1.3.7/pacparser-1.3.7.tar.gz"
 
+    version('1.3.8', sha256='4e2872de565b2b64ffc81ba503e0eba35b3f7ef4a023ddd4a328c7b9d2cac266',
+            url='https://github.com/manugarg/pacparser/releases/download/v1.3.8/pacparser-v1.3.8.tar.gz')
```
Comment (alalazo) [inline var/spack/repos/builtin/packages/pacparser/package.py]: Can we update the `url` and use a specific `url` in v1.3.7 ? Asking under the assumption that the new naming scheme is what will be used in the next versions.

**Review by alalazo — CHANGES_REQUESTED** [review]

File: var/spack/repos/builtin/packages/pacparser/package.py (hunk 1 above)
Comment (iarspider) [reply var/spack/repos/builtin/packages/pacparser/package.py]: Looks like new naming scheme will indeed be `v<version>`. Changed, added 1.3.9 (no URL, only a tag)

File: var/spack/repos/builtin/packages/pacparser/package.py [hunk 2]
```diff
@@ -9,14 +9,22 @@
...
 
+    maintainers = ['iarspider']
+
     homepage = "https://pacparser.github.io/"
-    url      = "https://github.com/manugarg/pacparser/releases/download/1.3.7/pacparser-1.3.7.tar.gz"
+    url      = "https://github.com/manugarg/pacparser/releases/download/v1.3.8/pacparser-1.3.8.tar.gz"
+    git      = "https://github.com/manugarg/pacparser.git"
 
-    version('1.3.7', sha256='eb48ec2fc202d12a4b882133048c7590329849f32c2285bc4dbe418f29aad249')
+    version('1.3.9', tag='v1.3.9')
```
Comment (alalazo) [inline var/spack/repos/builtin/packages/pacparser/package.py]: Can we use the corresponding commit sha?

Comment (iarspider) [reply var/spack/repos/builtin/packages/pacparser/package.py]: Ok.

**Review by alalazo — APPROVED** [review]

**Pull request merged**